playground/frontend/README.md

```diff
@@ -164,9 +164,12 @@ flutter test
 Integration tests currently can be run only on a local development machine.
 Server testing has not been verified yet.
 
-1. Install and run Chrome Driver: https://chromedriver.chromium.org/downloads
-2. Run it on port 4444: `chromedriver --port=4444`
-3. Run:
+1. Install Google Chrome: https://www.google.com/chrome/
+2. Install Chrome Driver: https://chromedriver.chromium.org/downloads
+   - Note: This GitHub action installs both Chrome and Chrome Driver:
+     https://github.com/nanasess/setup-chromedriver/blob/a249caaaad10fd12103028fd509853c2229eb6e6/lib/setup-chromedriver.sh
```

Comment:
FYI writing this comment for future reference in case we run into problems. The following worked for me on a Compute Engine Instance, matching the run job's `Ubuntu 22.04.1 LTS version`, and not the install google-chrome command in the [nanasess/setup-chromedriver](https://github.com/nanasess/setup-chromedriver/blob/a249caaaad10fd12103028fd509853c2229eb6e6/lib/setup-chromedriver.sh#L12) script.

On Ubuntu 22.04 (and on Debian not shown) the following error results from `sudo apt-get install -y google-chrome`
`E: Unable to locate package google-chrome`

Instead I did:
```
wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
sudo apt install ./google-chrome-stable_current_amd64.deb
```